Lib/ipaddress.py

```diff
@@ -1223,6 +1223,11 @@ def _parse_octet(cls, octet_str):
         if len(octet_str) > 3:
             msg = "At most 3 characters permitted in %r"
             raise ValueError(msg % octet_str)
+        # Handle leading zeros as strict as glibc's inet_pton()
+        # See security bug bpo-36384
+        if octet_str != '0' and octet_str[0] == '0':
+            msg = "Lead zeros are not permitted in %r"
```

Comment:
```suggestion
            msg = "Leading zeros are not permitted in %r"
```

Shouldn't it be 'leading'?

Reply:
fixed